src/crawler/index.ts

```diff
@@ -49,4 +49,6 @@ async function main(): Promise<void> {
   }, LOCATE_INTERVAL)
 }
 
-main().catch((err: Error) => log.error(`${err.message}`))
+if (typeof require !== 'undefined' && require.main === module) {
```

Comment:
Why is this required now?

Reply:
It was running when I imported from this file. This prevents that from happening (it's the JS equivalent of `if __name__ == '__main__'` in python).